test/php/gutenberg-coding-standards/Gutenberg/Tests/Commenting/FunctionCommentSinceTagUnitTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Unit test class for Gutenberg Coding Standard.
+ *
+ * @package gutenberg-coding-standards/gbc
+ * @link    https://github.com/WordPress/gutenberg
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace GutenbergCS\Gutenberg\Tests\Commenting;
+
+use PHP_CodeSniffer\Tests\Standards\AbstractSniffUnitTest;
+use PHP_CodeSniffer\Ruleset;
+
+/**
+ * Unit test class for the FunctionCommentSinceTagSniff sniff.
+ */
+final class FunctionCommentSinceTagUnitTest extends AbstractSniffUnitTest {
+
+	/**
+	 * Returns the lines where errors should occur.
+	 *
+	 * @return array <int line number> => <int number of errors>
+	 */
+	public function getErrorList() {
+		return array(
+			9  => 1,
```

Comment:
Maybe it would be good to document that only functions are covered at the moment.

It's annotated in the sniff, but it might not be immediately obvious when looking at the test in isolation.

Reply:
Fixed in 9bf6c1868644024f75f08d5b5a3da7ffbf614f4a.